config/routes.rb

```diff
@@ -6,8 +6,12 @@
   root 'users#index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   resources :users, only: [:index, :show] do
-  resources :posts, only: [:index, :show]
+  resources :posts, only: [:index, :new, :create, :show] do
+    resources :comments, only: [:create]
```

Comment:
- [ ] Kindly make sure you have a pair of routes in the `config/routes.rb` file for creating comments as required for the project i.e. you should also add a route for `new`.

- [ ] Once you have added a route for `new`, kindly make sure you have a **form** in the `views/comments/new.html.erb` file which accepts input for creating comments.